src/Raven.Client/ServerWide/DatabaseRecord.cs

```diff
@@ -193,6 +193,8 @@ public void AddIndex(IndexDefinition definition, string source, DateTime created
                 definition.ReduceOutputIndex = version;
             }
 
+            definition.ClusterState ??= new ClusterState();
+            definition.ClusterState.LastRollingDeploymentIndex = raftIndex;
```

Comment:
how do we know this is a rolling deployment? I see that there is a 'IsRolling' used few lines down

Reply:
moved